dockerfiles/feature-support/hsm-patches/0001-Introduce-generic-keypair-generation-interface-and-e.patch

```diff
@@ -474,6 +495,19 @@ index ec7f279..57cba43 100644
  /*
   * Store a private key on the token
   */
+diff --git a/src/p11_load.c b/src/p11_load.c
+index e89b0c7..3f69cec 100644
+--- a/src/p11_load.c
++++ b/src/p11_load.c
+@@ -81,7 +81,7 @@ int pkcs11_CTX_load(PKCS11_CTX *ctx, const char *name)
+ 	/* Tell the PKCS11 to initialize itself */
+ 	memset(&args, 0, sizeof(args));
+ 	/* Unconditionally say using OS locking primitives is OK */
+-	args.flags |= CKF_OS_LOCKING_OK;
++//	args.flags |= CKF_OS_LOCKING_OK;
```

Comment:
If we comment this out we can remove it entirely?

Reply:
yes